cmd/net_test.go

```diff
@@ -291,10 +291,9 @@ func TestSameLocalAddrs(t *testing.T) {
 		expectedErr error
 	}{
 		{"", "", false, errors.New("unable to process empty address")},
-		{":9000", ":9000", true, nil},
-		{"localhost:9000", ":9000", true, nil},
-		{"localhost:9000", "http://localhost:9000", true, nil},
-		{"8.8.8.8:9000", "http://localhost:9000", false, nil},
+		{"http://localhost:9000", ":9000", true, nil},
+		{"http://localhost:9000", "http://localhost:9000", true, nil},
+		{"http://8.8.8.8:9000", "http://localhost:9000", false, nil},
```

Comment:
> Why are these cases changed?

I had to adapt this because go 1.8 doesn't consider 8.8.8.8:9000 a network address anymore.

But still, it doesn't affect us, since

`minio server local_address:9000/export 8.8.8.8:9000/export 8.8.8.8:9000/export 8.8.8.8:9000/export`

is a standalone erasure and not distributed.